arch/src/x86_64/interrupts.rs

```diff
@@ -84,9 +84,8 @@ pub fn set_lint(vcpu: &VcpuFd) -> Result<()> {
 
 #[cfg(test)]
 mod tests {
-    extern crate rand;
-    use self::rand::Rng;
-
+    extern crate fc_util;
```

Comment:
`extern crate` isn't be needed anymore in Rust 2018.

Reply:
All these occurrences will get treated in the issue solving #839.